lib/bolt/project.rb

```diff
@@ -167,6 +166,25 @@ def modules
       @data['modules']
     end
 
+    # Returns the default modulepath for the project.
+    # If the 'modules' key is present in the project configuration file,
+    # use the new, shorter modulepath.
+    #
+    def modulepath
+      if modules
```

Comment:
This seems like it's really saying `unless modules.nil?`

Reply:
Tomato tomato?

Reply:
It's just more readable. This also seems like we should still be setting the attribute, either in this method as a `@modulepath ||=`, or in the init function. Recomputing it each time seems unnecessary..

Reply:
Given there's an `else`, `if` is clearer. `unless/else` is really confusing.